docs/apu_flashing_ipxe.md

```diff
@@ -0,0 +1,207 @@
+* IP of used PXE server: `192.168.0.108`
+
+Requirements
+------------
+
+* Ethernet connetion to the network
+* iPXE server have to be online in the same network
+* PXE boot have to be enabled in SeaBIOS options
+* OS with installed `flashrom`
+* Serial connection between APUx and PC
+
+Flashing procedure
+------------------
+
+1. Open APUx serial console. You can use `minicom` to do that.
+Parameters of connection:
+* Baudrate: 115200
+* Data bits: 8
+* Parity: None
+* Stop bits: 1
+
+```
+sudo minicom -b 115200 -o -D /dev/<name of APUx serial console visible in PC>
+```
+
+E.g.
+```
+sudo minicom -b 115200 -o -D /dev/ttyUSB0
+```
+
+2. Turn on APUx.
+
+3. When following communicate appears:
+```
+Press F10 key now for boot menu, N for PXE boot
+```
+press `N` to enter to the PXE boot menu.
+
+```
+   ---------------- iPXE boot menu ----------------
+   ipxe shell
+   autoboot
+```
+
+4. Select `ipxe shell`. You have to hurry because default option is `autoboot` and
+it will be selected after few seconds.
+
+5. After successful running the ipxe shell the following prompt will appear
+```
+iPXE> 
+```
+Now you have to type commands showed below:
+```
+dhcp net1
+set filename pxelinux.0
+set next-server 192.168.0.108
+chain tftp://${next-server}/${filename} 
+```
+
+> The `X` number in `netX` interface can be different depending on the connector 
+to which Ethernet is connected. If selected interface is connected to network
+information similar to the showed below should appear:
+```
+Configuring (net1 00:0d:b9:47:bb:e1).................. ok
+```
+> MAC address should be displayed. If there is no MAC address that means that
+there is no connection to the network for that interface.
+
+> IP placed next to `next-server` should be correct IP of used PXE server.
+
+6. After few seconds PXE server boot menu  should appear:
+
+```
+				 lqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqk
+				 x         PXE server boot menu          x
+				 tqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqu
+				 x Install                               x
+				 x Debian-netboot                        x
+				 x Voyage-netinst                        x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 x                                       x
+				 mqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqj
+
+				        Press [Tab] to edit options
+```
+
+Select `Debian-netboot` because it's the only OS with installed `flashrom`
+available in PXE server boot menu when instruction is being written. After that
+OS booting should start. 
+
+Number of OSes can be increased in the future.
+
+> For legacy versions of coreboot PXE server boot menu is incorrectly 
```

Comment:
not correct: character is doubled when `Legacy Console Redirection` is turned on is `sortbootorder`. It's because iPXE is outputting to the serial console and to the "screen" at the same time. "Screen" in this case is also serial console, due to `sgabios`, that is used to redirect the output to serial.

Reply:
I changed that. Is that information correct and sufficient now?